rust/chains/hyperlane-sealevel/src/validator_announce.rs

```diff
@@ -117,10 +118,10 @@ impl ValidatorAnnounce for SealevelValidatorAnnounce {
         _tx_gas_limit: Option<U256>,
     ) -> ChainResult<TxOutcome> {
         warn!(
-            "Announcing validator storage locations within the agents is not supported on Sealevel"
+            "Announcing validator storage locations within the agents is not supported on Sealevel"
```

Comment:
nit: any idea why the space shows a diff?

Reply:
Yeah, it was a nbsp and I replaced it with a normal ascii space